admin/test/features/support/page_objects/generate_pupil_pins_page.rb

```diff
@@ -5,11 +5,9 @@ class GeneratePupilPinsPage < SitePrism::Page
   element :generate_pin_message, '.column-two-thirds', text: 'Pupils will need a Personal Identification Number (PIN) and school password in order to start the check. These expire daily.'
   elements :info_message, '.list-number li'
   element :generate_pin_btn, 'input[value="Generate PINs"]'
-  element :generate_more_pin_btn, 'input[value="Generate more PINs"]'
-  element :school_password, '.column-two-thirds br'
 
-  element :select_all_pupils, '#selectAll'
-  element :unselect_all_pupils, '#selectAll'
+  element :select_all_pupils, '#tickAllCheckboxes'
+  element :unselect_all_pupils, '#tickAllCheckboxes'
```

Comment:
@sinhaamit1980 as its the same element, try getting a unique id via the text also, ie 
element :select_all_pupils, '#tickAllCheckboxes', text: 'Select all'

Reply:
Done